LayoutTests/animations/steps-transform-rendering-updates.html

```diff
@@ -37,8 +37,8 @@
             }, false);
             box.addEventListener('animationend', () => {
                 count = internals.renderingUpdateCount();
-                shouldBeTrue('count <= 9');
-                if (count > 9)
+                shouldBeTrue('count <= 10');
```

Comment:
How big can this get before indicating that the original fix is regressed?

Reply:
I added a comment to that effect.